src/config/dynamic-theme-fixes.config

```diff
@@ -2456,6 +2456,13 @@ CSS
 
 ================================
 
+buffer.com
+
+INVERT
+a[class*="BufferLogoWithWords"]
```

Comment:
```suggestion
IGNORE INLINE STYLE
div[class^="style__LogoWrapper"] *
```

Reply:
This was a learning experience. Thank you for the review!